docker/Dockerfile_agent

```diff
@@ -4,7 +4,8 @@ FROM ghcr.io/contrast-security-oss/vulneruby_engine/base:${RUBY_VER}
 
 ENV CI_TEST=true
 COPY agent/* agent/
-COPY contrast_security.yaml ./contrast_security.yaml
+RUN rm contrast_security.yaml || true
+COPY contrast_security.yaml contrast_security.yaml
 RUN gem install ./agent/contrast-agent.gem
```

Comment:
Just be sure there wasn't a prior one here. Really shouldn't matter and could revert if desired.